plugins/outputs/cloudwatch/cloudwatch.go

```diff
@@ -145,7 +145,7 @@ func (c *CloudWatch) Connect() error {
 		})
 
 	svc.Handlers.Build.PushBackNamed(handlers.NewRequestCompressionHandler([]string{opPutLogEvents, opPutMetricData}))
-	svc.Handlers.Build.PushBackNamed(handlers.NewCustomHeaderHandler("User-Agent", agentinfo.UserAgent()))
+	svc.Handlers.Build.PushBackNamed(handlers.NewCustomHeaderHandler("User-Agent", agentinfo.UserAgent("")))
```

Comment:
What are the repercussions of using an empty string for the group? What plugins are used in the empty string group?

Reply:
This is the `cloudwatch` output plugin instead of `cloudwatchlogs`. When I check this function/package, I don't see a variable for log group name, so I just put an empty string here. Is there a better variable I can use here?

Reply:
I don't think we need to worry about different goroutines for now as for `cloudwatch` plugin, the user agent will always be the same before and after this change:
- before: there was no group name as input so that it was always `agentinfo.userAgent`
- after: every thread/gorutine is using `""` as key to find `userAgentMap[""]`